### ISSUE TITLE
Schedule a regular Travis CI build with cron jobs

### ISSUE THREAD

**Issue by krichter722** [google/gson#1568]
Currently Travis CI master build fails 100% of time, see for an example https://www.travis-ci.org/krichter722/gson/builds/576977431. This could be detected early using Travis CI cron jobs as described at https://docs.travis-ci.com/user/cron-jobs/#adding-cron-jobs.

I might file a different issue for the build failure. Can you confirm this/trigger a master build?

**Comment by Montana** [comment]
Seems depending on the issue, Maven mixes the exception messages, but the actual problem is:
```bash
sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
```
Late response I know, but I hope this helps.

**Comment by krichter722** [comment]
@Montana Don't know what you're referring to. The build is passing regularly for a year now: https://travis-ci.org/github/google/gson/builds

**Comment by Marcono1234** [comment]
The original failure was fixed by #1607. Note that now Gson is built with GitHub Actions. It would probably still be possible to configure a scheduled build, but I am not sure if it would add that much value, except for making sure that the latest version of the specified JDK release can build the project.
